k8s/supportability/src/collect/logs/mod.rs

```diff
@@ -137,6 +137,28 @@ impl LogCollection {
 
             let mut hostname = None;
             if is_host_name_required(service_name.clone()) {
+                let status = pod.status.clone().ok_or_else(|| {
```

Comment:
```suggestion
                let status = pod.status.as_ref().ok_or_else(|| {
```

Reply:
removed the fetching status and `Pending` logic